wasm/prover/Cargo.toml

```
@@ -75,20 +75,11 @@ features = [
   'Response',
 ]
 
```

From my understanding we should also add the following line to enforce wasm32 feature flag is used and make it build with wasm
```suggestion
ring = { version = "0.17", features = ["wasm32_unknown_unknown_js"] }
```